projects/packages/assets/src/class-assets.php

```diff
@@ -498,14 +498,16 @@ public static function wp_default_scripts_hook( $wp_scripts ) {
 
 		// Can't use self::register_script(), this action is called too early.
 		if ( file_exists( __DIR__ . '/../build/i18n-loader.asset.php' ) ) {
-			$path  = '../build/i18n-loader.js';
-			$asset = require __DIR__ . '/../build/i18n-loader.asset.php';
+			$path     = '../build/i18n-loader.js';
+			$asset    = require __DIR__ . '/../build/i18n-loader.asset.php';
+			$jsx_path = '../build/react-jsx-runtime.js';
 		} else {
-			$path  = 'js/i18n-loader.js';
-			$asset = array(
+			$path     = 'js/i18n-loader.js';
+			$asset    = array(
 				'dependencies' => array( 'wp-i18n' ),
 				'version'      => filemtime( __DIR__ . "/$path" ),
 			);
+			$jsx_path = 'js/react-jsx-runtime.js';
```

Comment:
This file doesn't exist.

OTOH, if the package hasn't been built I don't know of anything that will work right. Even if we were to create this file, the browser wouldn't process the `import` or `require()` correctly. Even if we point it at `../node_modules/react/cjs/react-jsx-runtime.production.min.js`, that has a `require()` in it too.

So it might be best to just use `../build/react-jsx-runtime.js` always (just hard-code it on line 537 instead of setting a `$jsx_path` at all IMO) and let the browser 404 on that if the package hasn't been built. 🤷

Reply:
Good point, I've gone with hard coding: [ab51d24](https://github.com/Automattic/jetpack/pull/38428/commits/ab51d247bd562c505255199d237f80e18fba9b4f)